merchant_cheapest.py

```diff
@@ -0,0 +1,194 @@
+import argparse
+
+from pricewars import PricewarsMerchant, MerchantServer
+from pricewars.api import Marketplace, Producer
+from pricewars.models import Offer
+
+
+class CheapestMerchant(PricewarsMerchant):
```

Comment:
Since the register_and_start script has been removed: do you currently start the merchant automatically?

If not, it might be nice to have a parameter like `--start_immediately` which does that.

Reply:
Let's go over the changes next week. I don't feel comfortable accepting so many changes without really knowing what has been done.